tests/test_task_helpers.py

```diff
@@ -21,11 +22,21 @@ def test_get_previous_task_name_without_previous_task():
 
 def test_get_return_value_from_previous_task(context):
     expected = 'previous_return_value'
-    context['task_instance'].xcom_pull = lambda **kwargs: expected
+    add_return_value_from_previous_task(expected, context)
     result = get_return_value_from_previous_task(context)
     assert result == expected
 
 
+def test_get_return_value_from_branched_previous_task(branched_context):
+    expected = 'previous_return_value'
+    add_return_value_from_previous_task(expected, branched_context, task_id='branched_previous_task')
```

Comment:
should it be named `returning_previous_task` or similar, rather than `branched_previous_task`? Maybe this name is abstract and reusable across tests...

Reply:
would still keep the two branched task names similar, since there's no preferred one